File: Python/10886.py
import sys

f = lambda : sys.stdin.readline().rstrip()

n = int(f())
one = list()
zero = list()
for i in range(n):
    value = int(f())
    if value == 1:
        one.append(1)
    else:
        zero.append(0)
if len(zero) > len(one):
    print("Junhee is not cute!")
else:
    print("Junhee is cute!")

File: Java/_10886.java
package Baekjoon;

import java.io.BufferedReader;
import java.io.InputStreamReader;

public class _10886 {

    public static void main(String[] args){

        try {

            BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
            int num = Integer.parseInt(br.readLine());
            int cute = 0;
            for(int i = 0; i < num; i++){
                int value = Integer.parseInt(br.readLine());
                if(value == 0) cute -= 1;
                else cute += 1;
            }

            System.out.println(cute < 0 ? "Junhee is not cute!":"Junhee is cute!");

        } catch (Exception e) { }
    }
}
